File: app/src/main/java/com/example/syl/pokedex/datasource/PokemonService.java
package com.example.syl.pokedex.datasource;

import com.example.syl.pokedex.global.model.Pokemon;

import retrofit2.Call;
import retrofit2.http.GET;

public interface PokemonService {

    @GET("1")
    Call<Pokemon> getRandomPokemon();
}

File: app/src/main/java/com/example/syl/pokedex/datasource/PokemonService.kt
package com.example.syl.pokedex.datasource

import com.example.syl.pokedex.datasource.api.PokemonApiEntry

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface PokemonService {

    @GET
    fun getPokemon(@Url num: String): Call<PokemonApiEntry>
}